### ISSUE TITLE
 set Crow_FOUND to FALSE so package "Crow" is considered to be NOT   FOUND.

### ISSUE THREAD

**Issue by akjt** [CrowCpp/Crow#512]
I build crow using the guidline described here: 

https://crowcpp.org/master/getting_started/setup/linux/#using-cmake_1

I really dont understand when including crow in my cmake project I get the following: 

  Found package configuration file:

    /usr/local/lib/cmake/Crow/CrowConfig.cmake

  but it set Crow_FOUND to FALSE so package "Crow" is considered to be NOT
  FOUND.
  
  
  
  Why would have such thing set to False anyway and how to get rid of it.

**Comment by luca-schlecker** [comment]
I will take a look at it, it will take a bit though.
Are you still able to use Crow even though that variable is wrong?

**Comment by akjt** [comment]
Yes I am using it by just adding the sub_directory which the crow repo. Although, in the long run I would like to have the other option

**Comment by luca-schlecker** [comment]
I currently cannot confirm this behavior. Could you provide additional information?

I tried replicating with the following steps provided on Crow's `Getting Started` page:
[Installation instructions](https://crowcpp.org/master/getting_started/setup/linux/#using-cmake)
[Consumption instructions](https://crowcpp.org/master/getting_started/setup/linux/#using-cmake_1)

**Comment by gittiver** [comment]
Closed as no further information was provided for more than a year.
